confirm/confirm.go

```diff
@@ -0,0 +1,36 @@
+package confirm
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/fatih/color"
+)
+
+// AskForConfirmation asks the user for confirmation. A user must type in "yes" or "no" and
+// then press enter. It has fuzzy matching, so "y", "Y", "yes", "YES", and "Yes" all count as
+// confirmations. If the input is not recognized, it will ask again. The function does not return
+// until it gets a valid response from the user.
+func AskForConfirmation(s string) bool {
```

Comment:
This is a small nit-pick, but when reading confim.AskForConfirmation it stutters a little, I'd personally like to see confirm.Ask here, but it's down to you!

```suggestion
func Ask(s string) bool {
```

Reply:
Done